src/ItemInterface.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace MyOnlineStore\Omnipay\KlarnaCheckout;
+
+interface ItemInterface extends \Omnipay\Common\ItemInterface
+{
+    /**
+     * @return int
```

Comment:
Would prefer an additional description for these methods, similar to the original interface. Is this the amount of tax applied or the base amount over which tax is applied?